modules/cmdrunner/settings.go

```diff
@@ -33,5 +37,7 @@ func NewSettingsFromYAML(name string, moduleConfig *config.Config, globalConfig
 		maxLines: moduleConfig.UInt("maxLines", 256),
 	}
 
+	settings.width, settings.height = utils.CalculateDimensions(moduleConfig, globalConfig)
```

Comment:
If we calculate these on start, is it accounted for if we resize the window, or does that not matter?

Reply:
AFAIK the tool does not currently support resizing. Please correct me if I'm wrong.

Reply:
Widget dimensions can be modified in the `config.yml` file while the app is running, and it'll resize the affected widget accordingly.

Reply:
From what I read in the code basically the whole application is restarted. Can a widget somehow detect this?